spec/features/quotes/index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'Quote Index Page' do
+  it 'displays the total number returned by the search' do
+    visit root_path
+
+    fill_in :keyword, with: 'shadows'
+
+    click_on 'Search for Quotes'
+
+    expect(current_path).to eq(quotes_path)
+    expect(page).to have_content('Search Results')
+    expect(page).to have_content('Total Count: 6')
+    expect(page).to have_content('Results on this page: 6')
+    expect(page).to have_content('Quote Matches')
+  end
+
+  it 'displays 10 quotes that match the search' do
+    visit root_path
+
+    fill_in :keyword, with: 'shadows'
+
+    click_on 'Search for Quotes'
+
+    expect(page).to have_content('Author: Abraham Lincoln')
```

Comment:
I would probably put this tests expects into a `within` block. 
```
within(first(".quotes")) do
   expect(page).to have_content("Author: Rabbi Hillel")
   expect(page).to have_content("Quote: If I am not for myself, who will be for me? If I am not for others, what am I? And if not now, when?")
   expect(page).to have_content("Category(ies): famous-quotes, wisdom")
end
```
You can name it whatever you choose, it will just be looking for the `div class` or `div id` in your view.